Makefile

```
@@ -67,9 +67,9 @@ remove-tmp:
 	rm -rf ./tmp
 	rm -rf ./.vagrant
 	rm -rf ./.minio.sys
-	rm -rf ./example/.terraform
-	rm -rf ./example/terraform.tfstate
-	rm -rf ./example/terraform.tfstate.backup
+	rm -rf ./example/presto_cluster/.terraform
+	rm -rf ./example/presto_cluster/terraform.tfstate
+	rm -rf ./example/presto_cluster/terraform.tfstate.backup
```

```suggestion
	rm -rf ./example/*/.terraform
	rm -rf ./example/*/terraform.tfstate
	rm -rf ./example/*/terraform.tfstate.backup
```
Btw, this might be a better way if we include more examples in the future.

---

Thats a very good idea, so I added it!